### PR TITLE
Add license to be more explicit about MIT

### PR DESCRIPTION
**Pull request by MadLittleMods**
Add license to be more explicit about MIT

There is a MIT badge on the readme but we were slightly unclear if that was enough, https://gitlab.com/gitlab-org/gitlab-pages/-/merge_requests/336#note_409856646

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+The above copyright notice and this permission notice shall be
+included in all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED 'AS IS', WITHOUT WARRANTY OF ANY KIND,
+EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY
+CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
```
Comment (MadLittleMods) [inline LICENSE:22]: I copied this from https://github.com/tj/go-naturaldate/blob/master/LICENSE and changed the date to `2020`

**Comment by MadLittleMods** [comment]
Heya @tj, friendly ping for a review/merge!

**Comment by tj** [comment]
thanks!

**Comment by MadLittleMods** [comment]
Thanks for the great library @tj 😃 

Appreciate the quick merge! ❤️

**Pull request merged**